log

```diff
@@ -0,0 +1,343 @@
+		+   route_table_id = "${aws_route_table.privateweave-example-com.id}"
+		-   route_table_id = "${aws_route_table.private-us-test-1a-privateweave-example-com.id}"
+		    destination_cidr_block = "0.0.0.0/0"
+		+   gateway_id = "${aws_internet_gateway.privateweave-example-com.id}"
+		-   nat_gateway_id = "${aws_nat_gateway.us-test-1a-privateweave-example-com.id}"
+		  }
+		  
+		+ resource "aws_route" "private-us-test-1a-0-0-0-0--0" {
+		- resource "aws_route" "wan" {
+		+   route_table_id = "${aws_route_table.private-us-test-1a-privateweave-example-com.id}"
+		-   route_table_id = "${aws_route_table.main-privateweave-example-com.id}"
+		    destination_cidr_block = "0.0.0.0/0"
+		+   nat_gateway_id = "${aws_nat_gateway.us-test-1a-privateweave-example-com.id}"
+		-   gateway_id = "${aws_internet_gateway.privateweave-example-com.id}"
+		  }
+		  
+		...
+		  }
+		  
+		+ resource "aws_route_table" "private-us-test-1a-privateweave-example-com" {
+		- resource "aws_route_table" "main-privateweave-example-com" {
+		    vpc_id = "${aws_vpc.privateweave-example-com.id}"
+		    tags = {
+		      KubernetesCluster = "privateweave.example.com"
+		+     Name = "private-us-test-1a.privateweave.example.com"
+		-     Name = "main-privateweave.example.com"
+		    }
+		  }
+		  
+		+ resource "aws_route_table" "privateweave-example-com" {
+		- resource "aws_route_table" "private-us-test-1a-privateweave-example-com" {
+		    vpc_id = "${aws_vpc.privateweave-example-com.id}"
+		    tags = {
+		      KubernetesCluster = "privateweave.example.com"
+		+     Name = "privateweave.example.com"
+		-     Name = "private-us-test-1a.privateweave.example.com"
+		    }
+		  }
+		  
+		+ resource "aws_route_table_association" "private-us-test-1a-privateweave-example-com" {
+		- resource "aws_route_table_association" "main-us-test-1a-privateweave-example-com" {
+		+   subnet_id = "${aws_subnet.us-test-1a-privateweave-example-com.id}"
+		-   subnet_id = "${aws_subnet.utility-us-test-1a-privateweave-example-com.id}"
+		+   route_table_id = "${aws_route_table.private-us-test-1a-privateweave-example-com.id}"
+		-   route_table_id = "${aws_route_table.main-privateweave-example-com.id}"
+		  }
+		  
+		+ resource "aws_route_table_association" "utility-us-test-1a-privateweave-example-com" {
+		- resource "aws_route_table_association" "private-us-test-1a-privateweave-example-com" {
+		+   subnet_id = "${aws_subnet.utility-us-test-1a-privateweave-example-com.id}"
+		-   subnet_id = "${aws_subnet.private-us-test-1a-privateweave-example-com.id}"
+		+   route_table_id = "${aws_route_table.privateweave-example-com.id}"
+		-   route_table_id = "${aws_route_table.private-us-test-1a-privateweave-example-com.id}"
+		  }
+		  
+		...
+		    name = "api-elb.privateweave.example.com"
+		    vpc_id = "${aws_vpc.privateweave-example-com.id}"
+		+   description = "Security group for ELB in front of API"
+		-   description = "Security group for api ELB"
```

Comment:
We now have APIAccess and SSHAccess.  SSHAccess will apply to SSH (i.e. bastions), APIAccess will apply to the API (i.e. the ELB).  Is that what you mean?